lib/ansible/modules/system/group.py

```diff
@@ -445,7 +470,8 @@ def main():
             name=dict(type='str', required=True),
             gid=dict(type='int'),
             system=dict(type='bool', default=False),
-            local=dict(type='bool', default=False)
+            local=dict(type='bool', default=False),
+            non_unique=dict(type='bool', default=False)
```

Comment:
Please add a comma to the last element. This makes diffs easier to read and makes syntax errors less likely when adding or reordering items.
```suggestion
            non_unique=dict(type='bool', default=False),
```

Reply:
Done!